src/goto-instrument/Makefile

```diff
@@ -94,12 +94,12 @@ INCLUDES= -I ..
 
 LIBS =
 
-CLEANFILES = goto-instrument$(EXEEXT)
+CLEANFILES = goto-instrument$(EXEEXT) goto-instrument$(LIBEXT)
```

Comment:
Perhaps the intent was already there, but this should be squashed with the commit that introduces the unit tests so that they compile.